prac_06/guitars.py

```diff
@@ -0,0 +1,29 @@
+from prac_06.guitar import Guitar
```

Comment:
I noticed that there are a number of your files with no doc strings at the start. Might it be good to add some to inform someone what the code does?